src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Builds.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Immutable;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    public class ScenarioTests_Builds : MaestroScenarioTestBase
+    {
+        private string repoUrl;
+        private readonly string repoName = "maestro-test1";
+        private readonly string sourceBuildNumber = "654321";
+        private readonly string sourceCommit = "123456";
+        private readonly string sourceBranch = "master";
+        private readonly IImmutableList<AssetData> sourceAssets = ImmutableList<AssetData>.Empty;
```

Comment:
It's odd to assign this readonly field twice... once here and once in the constructor.  I'd remove this initializer, otherwise someone might think a readonly immutable list wouldn't change. :-)